api/sixtyfps-cpp/docs/cpp_migration.md

```diff
@@ -29,3 +29,19 @@ New code:
 sixtyfps::Value args[] = { SharedString("Hello"), 42. };
 instance->invoke_callback("foo", args);
 ```
+
+#### Models
+
+`Model::row_data` will now return a `std::optional<ModelData>` and can thus be used with indices that are out of bounds.
+
+Old code:
+
+```cpp
+std::get<2>(another_model->row_data(2)), 336.);
+```
+
+New code:
+
+```cpp
+std::get<2>(*another_model->row_data(2)), 336.); // Note the extra *. You should check whether the returned value `has_value()` before accessing it
```

Comment:
```suggestion
```cpp
// `another_model` is a model that contains floats.
std::optional<float> entry = another_model->row_data(2);
if (entry.has_value()) {
    do_something(*entry);
} else {
    // row index 2 is out of bounds
}
```